pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -1882,6 +1904,10 @@ def pow(self, z):
     def control_wires(self):
         return Wires(self.wires[:2])
 
+    @property
+    def is_hermitian(self):
+        return True
+
 
 class MultiControlledX(Operation):
```

Comment:
`MultiControlledX` should be Hermitian too.

Reply:
Done!